src/main/webapp/mentors.js

```diff
@@ -55,81 +105,173 @@ function createMentorElement(mentor) {
+ * @param {String} img
+ * @param {HTMLElement} mentorCardBody
+ * @return {HTMLElement} mentorCardBody
+ */
+function addImg(img, mentorCardBody) {
+  if (img == null) {
+    return mentorCardBody;
+  }
+  const mentorImg = document.createElement('img');
+  mentorImg.className = 'mentor-picture';
+  mentorImg.src = img;
+  mentorCardBody.appendChild(mentorImg);
+  return mentorCardBody;
+}
 
-  mentorCard.appendChild(mentorCardBody);
-  mentorContainer.appendChild(mentorCard);
+/**
+ * Creates a description for mentor card.
+ * @param {String} bio
+ * @param {HTMLElement} mentorCardBody
+ * @return {HTMLElement} mentorCardBody
+ */
+function addBio(bio, mentorCardBody) {
+  if (bio == null) {
+    return mentorCardBody;
+  }
+  const mentorDescription = document.createElement('p');
+  mentorDescription.innerHTML = bio;
+  mentorCardBody.appendChild(mentorDescription);
+  return mentorCardBody;
+}
 
-  return mentorContainer;
+/**
+ * Creates a name tag for mentor card.
+ * @param {String} name
+ * @param {HTMLElement} mentorCardBody
+ * @return {HTMLElement} mentorCardBody
+ */
+function addName(name, mentorCardBody) {
+  if (name == null) {
+    return mentorCardBody;
+  }
+  const mentorName = document.createElement('h5');
+  mentorName.className = 'card-title text-primary';
+  mentorName.innerHTML = name;
+  mentorCardBody.appendChild(mentorName);
+  return mentorCardBody;
 }
 
 /**
  * Creates a github profile button element from username.
  * @param {String} gitHubID
- * @return {HTMLElement} userGithubButton
+ * @param {HTMLElement} mentorCardBody
+ * @return {HTMLElement} mentorCardBody
  */
-function createGitHubLink(gitHubID) {
+function createGitHubLink(gitHubID, mentorCardBody) {
+  if (gitHubID == null) {
+    return mentorCardBody;
+  }
   const gitHubBaseUrl = 'https://github.com/';
   const gitLink = gitHubBaseUrl.concat(gitHubID);
   const userGithubButton = document.createElement('a');
   userGithubButton.innerText = 'GitHub Profile';
   userGithubButton.className = 'btn btn-primary';
   userGithubButton.role = 'button';
   userGithubButton.href = gitLink;
-  return userGithubButton;
+  mentorCardBody.appendChild(userGithubButton);
+  return mentorCardBody;
 }
 
 /**
  * Creates a block of interest tags.
  * @param {String[]} interestTags
- * @return {HTMLElement} tagDiv
+ * @param {HTMLElement} mentorCardBody
+ * @return {HTMLElement} mentorCardBody
  */
-function createTagDiv(interestTags) {
+function createTagDiv(interestTags, mentorCardBody) {
+  if (interestTags == null) {
+    return mentorCardBody;
+  }
   const tagDiv = document.createElement('div');
-  tagDiv.className = 'row p-3';
+  tagDiv.className = 'd-flex justify-content-center p-3';
   for (const tagText of interestTags) {
     const tagElement = document.createElement('div');
-    tagElement.className = 'border border-muted text-muted mr-1 mb-1 badge';
+    tagElement.className = 'border border-muted text-muted mr-1 mb-1 badge text-center';
```

Comment:
ESLint line length fix here
```suggestion
    tagElement.className = 
        'border border-muted text-muted mr-1 mb-1 badge text-center';
```